rosa_architecture/rosa-architecture-models.adoc

```diff
@@ -0,0 +1,14 @@
+[id="rosa-architecture-models"]
+= Architecture models
```

Comment:
Add ROSA to the heading

Reply:
If I do that, I'd have to change the TOC entry as well. And I hesitate to do that because we want to mirror OSD as much as possible. I'm open to talking about this, but am leaving it for now.

Reply:
I'm thinking in terms of browser search and finding this topic. Ok to leave, of course. We'll need to take a comprehensive look at all the doc.